disneyApi/src/components/Modal/Modal.jsx

```diff
@@ -0,0 +1,88 @@
+import { useContext } from 'react';
+import { ThemeContext } from '../../context/ThemeContext';
+import {
+	ModalImg,
+	ModalList,
+	ModalLists,
+	ModalContent,
+	ModalBackground,
+} from './';
+import { Button } from '../';
+import useEscKey from '../../hooks/useEscKey';
+
+const Modal = ({ setIsModalShown, data }) => {
+	const { themeColors } = useContext(ThemeContext);
+
+	const dataEntries = Object.entries(data);
+	const filteredDataEntries = dataEntries.filter(
```

Comment:
The filter condition can be simplified by using an array and includes method for better readability.
 (item) =>['films', 'shortFilms', 'videoGames', 'tvShows'].includes(item[0]) && item[1].length